src/coreclr/nativeaot/Runtime/EtwEvents.h

```diff
@@ -833,33 +825,17 @@ TemplateEventDescriptor(REGHANDLE RegHandle, const EVENT_DESCRIPTOR * Descriptor
 
 #define ETW_EVENT_ENABLED(Context, EventDescriptor) false
 
-#define FireEtwBGC1stConEnd(ClrInstanceID)
-#define FireEtwBGC1stNonConEnd(ClrInstanceID)
-#define FireEtwBGC2ndConBegin(ClrInstanceID)
-#define FireEtwBGC2ndConEnd(ClrInstanceID)
-#define FireEtwBGC2ndNonConBegin(ClrInstanceID)
-#define FireEtwBGC2ndNonConEnd(ClrInstanceID)
-#define FireEtwBGCAllocWaitBegin(Reason, ClrInstanceID)
-#define FireEtwBGCAllocWaitEnd(Reason, ClrInstanceID)
-#define FireEtwBGCBegin(ClrInstanceID)
-#define FireEtwBGCDrainMark(Objects, ClrInstanceID)
-#define FireEtwBGCOverflow(Min, Max, Objects, IsLarge, ClrInstanceID)
 #define FireEtwBGCPlanEnd(ClrInstanceID)
-#define FireEtwBGCRevisit(Pages, Objects, IsLarge, ClrInstanceID)
 #define FireEtwBGCSweepEnd(ClrInstanceID)
-#define FireEtwGCFullNotify_V1(GenNumber, IsAlloc, ClrInstanceID)
 #define FireEtwGCGlobalHeapHistory_V1(FinalYoungestDesired, NumHeaps, CondemnedGeneration, Gen0ReductionCount, Reason, GlobalMechanisms, ClrInstanceID)
 #define FireEtwGCJoin_V1(Heap, JoinTime, JoinType, ClrInstanceID)
 #define FireEtwGCOptimized_V1(DesiredAllocation, NewAllocation, GenerationNumber, ClrInstanceID)
 #define FireEtwGCPerHeapHistory()
 #define FireEtwGCSettings(SegmentSize, LargeObjectSegmentSize, ServerGC)
-#define FireEtwPinPlugAtGCTime(PlugStart, PlugEnd, GapBeforeSize, ClrInstanceID)
-#define FireEtwPrvDestroyGCHandle(HandleID, ClrInstanceID)
 #define FireEtwPrvGCMarkCards_V1(HeapNum, ClrInstanceID)
 #define FireEtwPrvGCMarkFinalizeQueueRoots_V1(HeapNum, ClrInstanceID)
 #define FireEtwPrvGCMarkHandles_V1(HeapNum, ClrInstanceID)
 #define FireEtwPrvGCMarkStackRoots_V1(HeapNum, ClrInstanceID)
-#define FireEtwPrvSetGCHandle(HandleID, ObjectID, Kind, Generation, AppDomainID, ClrInstanceID)
 
```

Comment:
Is it possible to add some comments on the FireEtw* and FireEtXplat* mix in the !FEATURE_ETW please?

Reply:
The only one actually used is `FireEtwGCSettings`. I'll delete the rest and move that one over to be like the others (FireEtXplat - which actually fires ETW - in here, FireEtw at the disabled/enabled boundary).